homeassistant/components/airgradient/iqs.json

```diff
@@ -0,0 +1,5 @@
+{
+  "IQS001": {
+    "implemented": true
+  }
+}
```

Comment:
So a few thought here.
1. Something is either implemented or it isn't. I do like the idea of having a shorthand notation like
```json
{
  "IQS001": true
}
```
But I can see that it is a bit inconsistent.
2. I don't feel like creating a file with every rule added to `false`. Although that might be a good way to update it quickly without a lot of copy and paste. So I am a bit 50/50 on that. We could also ask users to add as many rules until their next tier, so every integration would just get a set of bronze rules assigned to `false`. But this might demotivate people to score higher.
3. Ideally, the IQS are sorted at 1-2-3, but the downside is, when looking at the docs, we show them like bronze 1-2-4, silver 3-5-6. So sorting them 1-2-3, we would create 1 list with all tiers used through each other. Should we store them per tier? (Downside: if we ever change the tier of a rule, that causes a big change in the files, upside: its more clear).

Reply:
Would this work?
```json
{
  "IQS001": {
    "status": "implemented"
  }
}
```
```json
{
  "IQS001": {
    "status": "exempt",
    "reason": "it doesn't apply"
  }
}
```

Reply:
Another option to consider is some of the linters use a format like this:

```
include:
- IQS001
- IQS004
- IQS005

exclude:
# Does not control devices
- IQS002
- IQS003
```
Though probably not those names

Reply:
It would seem to me a bit unnecessary to have a reason code when it can be just a comment on that line.
I'd like @allenporter suggestion but maybe implemented/excepted  instead of include/exclude.